tests/phpunit/AutoReview/Makefile/MakefileTest.php

```diff
@@ -62,9 +63,15 @@ final class MakefileTest extends TestCase
 
     public function test_the_default_goal_is_the_help_command(): void
     {
+        try {
+            shell_exec('command -v timeout');
+            $timeout = 'timeout 2s';
+        } catch (ExecException) {
+            $timeout = '';
+        }
         $output = shell_exec(sprintf(
             '%s make --silent --file %s 2>&1',
-            shell_exec('command -v timeout') !== null ? 'timeout 2s' : '',
```

Comment:
And here, but, arguably, the new variant is easier to read.

Reply:
I left that one in :)